PIconnect/AFSDK.py

```diff
@@ -184,6 +184,7 @@ class PISystems:
             """
 
             DefaultPISystem = None
+            Version = "0.0.0.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/manual/Hugovdberg/PIconnect/pullRequest?prid=5143034)